app/assets/javascripts/passengers/passengers.js

```diff
@@ -1,4 +1,19 @@
 $( document ).on("turbolinks:load", function() {
+  var clipboard = new ClipboardJS('#copybtn', {
+    text: function() {
+      return $("#passengers tbody tr").map(function(){
+        return $(this).data("email")
+      }).get().join(";")
+    }
+  });
+  clipboard.on('success', function() {
+    $('#copybtn').tooltip({title: "Copied"}).tooltip('show');
+  });
+
+  clipboard.on('error', function() {
```

Comment:
I couldn't find a way to merge this into one block.

Reply:
Yeah, all the ways I could think of would make Code Climate happier, but actual developers sadder. For instance:

```js
  var clipResults = {'success': 'Copied', 'error': 'Falied to copy!'}
  for (const event in clipResults) {
    if (!clipResults.hasOwnProperty(event)) continue;
    clipboard.on(event, function() {
      $('#copybtn').tooltip({title: clipResults[event]}).tooltip('show');
    });
  }
```

But it: a) Requires ES5+, b) is the same number of lines of code, and c) adds a level of inderection that makes the code _harder_ to understand.